src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Azs.Backup.Admin.psd1

```diff
@@ -131,20 +131,10 @@ Changes may cause incorrect behavior and will be lost if the code is regenerated
             # IconUri = ''
 
             # ReleaseNotes of this module
-            ReleaseNotes = '## 2018.9.10
-            * Module dependencies updated
-                * AzureRM.Profile
-                * AzureRM.Resources
-            * Support handling names of nested resources
-                * Get-AzsBackup
-                * Restore-AzsBackup
-                * Start-AzsBackup
-            * Deprecations
-                * Set-AzsBackupShare is an alias now to the cmdlet Set-AzsBackupConfiguration
-                * Get-AzsBackupLocation is an alias now to the cmdlet Get-AzsBackupConfiguration
-                * Set-AzsBackupConfiguration, the parameter BackupShare is an alias now for the parameter path
-            * Bug fixes
-                * Handle ErrrorAction correctly now
+            ReleaseNotes = '## 2019.01.04
+            * Backup changes to cert-based encryption mode
```

Comment:
Explicltly call out  the breaking changes, showing the old usage vs new usage would be helpful

Reply:
Added. Thanks.